core/rfb.js

```diff
@@ -1543,6 +1872,90 @@
             sock.flush();
         },
 
+        atenKeyEvent: function (sock, keysym, down) {
+            var ks = XK2HID[keysym];
+            var buff = sock._sQ;
+            var offset = sock._sQlen;
+
+            buff[offset] = 4;
+            buff[offset + 1] = 0;
+            buff[offset + 2] = down;
+
+            buff[offset + 3] = 0;
+            buff[offset + 4] = 0;
+            
+            buff[offset + 5] = (ks >> 24);
+            buff[offset + 6] = (ks >> 16);
+            buff[offset + 7] = (ks >> 8);
+            buff[offset + 8] = ks;
+
+            buff[offset + 9] = 0;
+            buff[offset + 10] = 0;
+            buff[offset + 11] = 0;
+            buff[offset + 12] = 0;
+
+            buff[offset + 13] = 0;
+            buff[offset + 14] = 0;
+            buff[offset + 15] = 0;
+            buff[offset + 16] = 0;
+
+            buff[offset + 17] = 0;
+
+            sock._sQlen += 18;
+        },
+
+        atenPointerEvent: function (sock, x, y, mask) {
```

Comment:
I can't say for sure until I locate my notes, but I'm fairly confident that it would be ignored.  I don't recall seeing anything related to multiple types of key or pointer events in the iKVM server.